tutorials/admin/troubleshooting/faq.md

```diff
@@ -0,0 +1,18 @@
+# Frequently Asked Questions
+
```

Comment:
Most FAQs have a list of questions at the top that link to the question and its discussion further down the page. This is a very helpful navigation feature. I think we should include a table of contents here.